tools/shell/shell.cpp

```diff
@@ -4695,6 +4695,7 @@ static const char zOptions[] = "   -ascii               set output mode to 'asci
                                "   -separator SEP       set output column separator. Default: '|'\n"
                                "   -table               set output mode to 'table'\n"
                                "   -unredacted          allow printing unredacted secrets\n"
+                               "   -ui                  open a web interface\n"
```

Comment:
What about to mention it loads UI extension also? Something like `load UI extension and starts web interface`.

Reply:
I understand what you mean, but I want to point out that that behavior is configurable.
```c++
	//! The command to execute when `-ui` is passed in
	string ui_command = "CALL start_ui()";
```

In the meta commands:
```c++
{"ui_command", 0, SetUICommand, "[command]", "Set the UI command", 0},
```

Combined with the `-cmd` / `-c` option, this can be provided on startup.

`start_ui` is registered as part of the `ui` extension, which triggers autoinstall/autoload if enabled.
(from `extension_entries.hpp`)
```c++
{"start_ui", "ui", CatalogType::TABLE_FUNCTION_ENTRY},
```

If anything the help on the `-ui` command should be more abstract, because all it really does is:
> execute the command registered with `.ui_command` (default `CALL start_ui()`)

Reply:
How about:

```
Launches a web interface using the `ui` extension. This behavior can be configured using the `.ui_command` option.
```